src/MusicTheory/PitchClass.elm

```diff
@@ -241,8 +152,8 @@ transposeDown interval pc =
 
 -}
 transposeBySemitones : Int -> PitchClass -> PitchClass
```

Comment:
It doesn't make sense to me to transpose by semitones. You can really only transpose by intervals. Why do we have this?